haas/ext/switches/brocade.py

```diff
@@ -60,33 +60,41 @@ def session(self):
     def disconnect(self):
         pass
 
-    def apply_networking(self, action):
+    def modify_port(self, port, channel, network_id):
         """ Apply a NetworkingAction to the switch.
 
         Args:
             action: NetworkingAction to apply to the switch.
         """
-        interface = action.nic.port.label
-        channel = action.channel
+        # XXX: We ought to be able to do a Port.query ... one() here, but
```

Comment:
FIXME is typically used for something that's actually *broken.* as far as I know this code is *correct,* It's just not the most elegant, so I think XXX is more appropriate. I will open an issue for this. I want to wait until this is merged though so I can link to section of the code in the merge commit, rather than some potentially transient state on my branch.